packages/client/components/ActivityLibrary/ActivityCard.tsx

```diff
@@ -72,6 +109,17 @@ export const ActivityCard = (props: ActivityCardProps) => {
           </div>
         </div>
       )}
+      {template && (
+        <Tooltip open={showTooltip}>
+          <TooltipTrigger asChild>
+            <div className='h-50 w-50 text-sky-500 hover:text-sky-700' />
+          </TooltipTrigger>
+          <TooltipContent className='max-w-md whitespace-normal rounded-lg bg-white p-4 text-left text-slate-700 shadow-lg hover:cursor-pointer sm:max-w-sm'>
```

Comment:
Sometimes tooltip cover the whole card, if I click on it I go to that template, which might be not obvious behaviour
Maybe clicking should only close tooltip or have pointer cursor

<img width="445" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/702d7273-523a-4083-8110-02a5350d4d01">